### PR TITLE
OMG, I think this might be the one

### PR DESCRIPTION
**Pull request by daytonn**
@kand @collin

### CODE REVIEW

File: elements/campaign-display/campaign-display-examples.js [hunk 1]
```diff
@@ -1,22 +1,38 @@
...
+      },
+    },
+
+    'CampaignDisplay (no name)': {
+      render() {
+        return `
+          <campaign-display
+            src="http://localhost:8080/fixtures/campaign-display/campaign.json"
+            preamble-text="Sponsored by"
+            image-only={true}
```
Comment (collin) [inline elements/campaign-display/campaign-display-examples.js]: should be `image-only`?

**Comment by collin** [comment]
👍

File: elements/campaign-display/campaign-display-examples.js (hunk 1 above)
Comment (daytonn) [inline elements/campaign-display/campaign-display-examples.js]: good catch

**Comment by kand** [comment]
👍

**Comment by kand** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;ktUIPjvB&#x2F;8-campaign-component-to-display-sponsored-content-from-tunic"> Campaign Component to Display Sponsored Content from Tunic</a>

**Pull request merged**